oci/charmed-opensearch/contacts.yaml

```diff
@@ -0,0 +1,4 @@
+# Who and how to notify about the build progress.
+notify:
+  mattermost-channels:  # These are channel IDs and not names.
+    - 5dom1psz9bbzfcff8kzdei7p5h  # data platform alerts
```

Comment:
pls double check that this is in Canonical's MM server

Reply:
This is a private channel in the Canonical MM's server, should it perhaps be a public one?

Reply:
That should be alright ;)